content/design-systems-for-developers/react/en/review.md

```diff
@@ -123,13 +135,15 @@ We’ll demo visual review by making a UI change on a new branch.
 git checkout -b improve-button
 ```
 
-First, tweak the Button component. “Make it pop” – our designers will love it.
+First, tweak the Button component by replacing the `border` and `font-size` CSS properties. “Make it pop” – our designers will love it.
 
 ```javascript
+// Button.js
 // ...
 const StyledButton = styled.button`
-  border: 10px solid red;
+  ... border: 10px solid red;
```

Comment:
```suggestion
  ...
  border: 10px solid red;
```

Reply:
Think it looks slightly clearer what is going on here if things are on different lines.